src/util/serverUtils.js

```diff
@@ -130,15 +113,12 @@ export function registerExtensionEvents(server) {
 /**
  * Make any environment changes that need to be made in response to the provided
  * config
- * @param {Object} config
- * @return {Object} the original config object
+ * @param {Object} config the node-convict config object
+ * @return null
  */
 export function configureEnv(config) {
 	// When using .dev.meetup endpoints, ignore self-signed SSL cert
-	const USING_DEV_ENDPOINTS = checkForDevUrl(config);
-	https.globalAgent.options.rejectUnauthorized = !USING_DEV_ENDPOINTS;
-
-	return config;
+	https.globalAgent.options.rejectUnauthorized = config.get('isProd');
```

Comment:
at the moment, NODE_ENV and api urls are independent, so `checkForDevUrl` could return a different value than `!config.get('isProd')`. Let's leave `checkForDevUrl` in place for now, and I'll clean it up in https://meetup.atlassian.net/browse/WP-158